module/VuFind/src/VuFind/AjaxHandler/AbstractUserRequestAction.php

```diff
@@ -67,14 +67,16 @@ public function handleRequest(Params $params)
         $requests = $this->ils->{$this->lookupMethod}($patron);
         $status = [
             'available' => 0,
-            'in_transit' => 0
+            'in_transit' => 0,
+            'other' => 0
         ];
         foreach ($requests as $request) {
-            if ($request['available'] ?? false) {
+            if (!empty($request['available'])) {
                 $status['available'] += 1;
-            }
-            if ($request['in_transit'] ?? false) {
+            } elseif (!empty($request['in_transit'])) {
                 $status['in_transit'] += 1;
+            } else {
+                $status['other'] += 1;
```

Comment:
Tiny nitpick, but we could use `++` instead of all these `+= 1` as a slight optimization. :-)